apiserver/remoterelations/remoterelations.go

```diff
@@ -184,55 +184,134 @@ func (api *RemoteRelationsAPI) RelationUnitSettings(relationUnits params.Relatio
 	return results, nil
 }
 
-// Relations returns information about the cross-model relations with the specified keys
-// in the local model.
-func (api *RemoteRelationsAPI) Relations(entities params.Entities) (params.RemoteRelationResults, error) {
-	results := params.RemoteRelationResults{
-		Results: make([]params.RemoteRelationResult, len(entities.Entities)),
+// IngressNetworksForRelation returns any CIDRs for which ingress is required to allow
```

Comment:
I've renamed the methods to IngressSubnetsForRelation.

I didn't see the need to use a Subnet struct instead of just a CIDR because by definition these subnets are provider specific and their use is to be passed directly to Open/ClosePorts() which construct an IngressRule from a port range and CIDR slice. In turn these rules set a CIDR slice on a cloud security group.

Is is possible that a cloud machine can be connected to different VPC subnets which have the same CIDR? Would all subnets on a machine have unique CIDRs? Not sure??